src/parallel-rustc.md

```diff
@@ -1,31 +1,35 @@
 # Parallel Compilation
 
-As of <!-- date-check --> August 2022, the only stage of the compiler that 
-is already parallel is codegen. Some parts of the compiler already have 
-parallel implementations, such as query evaluation, type check and 
-monomorphization, but the general version of the compiler does not include 
-these parallelization functions. **To try out the current parallel compiler**, 
-one can install rustc from source code with `parallel-compiler = true` in 
-the `config.toml`.
+As of <!-- date-check --> August 2022, the only stage of the compiler that is
+parallel is [code generation stage][codegen] (codegen). Some parts of the
+compiler have parallel implementations, such as query evaluation, type check
+and [monomorphization][monomorphization], but the general version of the
+compiler does not include parallelization functions. **To try out the current
+parallel compiler**, install `rustc` from source code with `parallel-compiler =
+true` in the `Config.toml`.
```

Comment:
cc @SparrowLii I feel like some of this is quite outdated, right?

Reply:
Most of the processes in the rust compiler have been parallelized.

The codegen part is executed concurrently by default.  Users can control the number of tasks with the `-C codegen-units` option.

The compilation parts after HIR lowering to codegen, such as type checking, borrow checking and mir optimization have been parallelized in the nightly version. Currently, they are executed in serial by default, and parallelization is manually enabled by the user using the `-Z threads = n` option.

Other parts such as lex parsing, HIR lowering, and macro expansion are still executed in serial mode.